Dockerfile

```diff
@@ -38,7 +38,7 @@ RUN apt-get update \
 		wget \
 	&& rm -rf /var/lib/apt/lists/*
 
-ENV NODE_VERSION 6.2.0
+ENV NODE_VERSION 6.2.2
 ENV NPM_VERSION 3.8.9
```

Comment:
actually, @Page- maybe you meant 3.10.1? https://github.com/npm/npm/blob/master/CHANGELOG.md#v3101-2016-06-17